recipes/bob.extension/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.0.11" %}
+
+package:
+  name: bob.extension
+  version: {{ version }}
+
+source:
+  md5: a2ccb5f5a5f0abee02488565e636441d
+  fn: bob.extension-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.extension/bob.extension-{{ version }}.zip
+
+build:
+  entry_points:
+  - bob_new_version.py = bob.extension.scripts:new_version
+  - bob_dependecy_graph.py = bob.extension.scripts:dependency_graph
+  number: 0
+  preserve_egg_dir: false
+
+requirements:
+  run:
+  - python
+  - setuptools
+  build:
```

Comment:
Could we please space out the build and run sections? Also, could we please flip the order?